wot-wg-2023-draft.html

```diff
@@ -296,7 +296,26 @@ <h2>Scope</h2>
           <dd>Define bindings for specific protocols and payloads of interest while focusing to support ecosystems.</dd>
           <dt><strong>Interoperability Profiles</strong>
             (<a href="#profiles-deliverable">Profiles</a>):</dt>
-          <dd>Support out-of-the-box interoperabilty via a profiling mechanism.</dd>
+          <dd>
+	    Support out-of-the-box interoperabilty via a profiling mechanism.
+	  </dd>
+	  <dt>
+	    <strong>Digital Twins</strong>
+            (<a href="#profiles-deliverable">Profiles</a>, 
+	     <a href="#thing-description-deliverable">Thing Description</a>):
+	  </dt>
+          <dd>
+	    Enable using WoT for describing digital twins, i.e. virtual things that are an abstraction of real world devices and services.
+	  </dd>
+	  <dt>
+	    <strong>Behavioral Description</strong>
+            (<a href="#profiles-deliverable">Profiles</a>, 
+	     <a href="#thing-description-deliverable">Thing Description</a>):
+	  </dt>
+          <dd>
+	    Formal description of the behavior of things, 
+	    thing models and digital twins.
```

Comment:
behavior of a thing model is a bit of a weird term given the current use of Thing Models

Reply:
changed the text.